### PR TITLE
config: Add CentOS Stream 9 + EPEL Next 9

### LINKED ISSUE #821: centos-stream+epel-9-next config is missing?
### Short description of the problem

Looking into https://github.com/rpm-software-management/mock/tree/main/mock-core-configs/etc/mock it seems that the `centos-stream+epel-9-next` configs were renamed to `centos-stream+epel-9`, but there should be both of them present. One with EPEL 9 and the other one with EPEL 9 Next. Also, the EPEL 9 Next configs should probably contain both Next and "no-Next" EPEL repos.

### PR DESCRIPTION
**Pull request by Conan-Kudo**
This allows people to build against EPEL Next 9.

Fixes #821.

### CODE REVIEW

**Comment by Conan-Kudo** [comment]
cc: @hroncok

File: mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+include('templates/centos-stream-9.tpl')
+include('templates/epel-9.tpl')
+include('templates/epel-next-9.tpl')
```
Comment (hroncok) [inline mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg:3]: All 2 EPEL templates have a local repo. Does the latter win, or how does that work?

Comment (Conan-Kudo) [reply mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg:3]: That's... a good question. I'm not sure what happens here, but in any case, we need to dedupe these references. We have _three_ `local` references across these templates. And that's true for both 8 and 9.

I'm tempted to rename the EPEL ones to `epel-local` and `epel-next-local`. We'd need to document this change somewhere so people know how to use it, though.

Comment (Conan-Kudo) [reply mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg:3]: That's... a good question. I'm not sure what happens here, but in any case, we need to dedupe these references. We have _three_ `local` references across these templates. And that's true for both 8 and 9.

I'm tempted to rename the EPEL ones to `epel-local` and `epel-next-local`. We'd need to document this change somewhere so people know how to use it, though.

@carlwgeorge, what do you think?

Comment (praiskup) [reply mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg:3]: Perhaps rename them to `local-epel` and `local-epel-next`, so users can just do `--enablerepo 'local*'`.

Comment (Conan-Kudo) [reply mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg:3]: Oh, that makes sense! I'll make a separate PR for that.

Comment (Conan-Kudo) [reply mock-core-configs/etc/mock/centos-stream+epel-next-9-aarch64.cfg:3]: Done in #824

**Comment by praiskup** [comment]
Do we want to have this in the very next mock-core-configs release, or can this wait another 14+ days?

**Comment by Conan-Kudo** [comment]
We need this and #824 for the next core-configs release. Right now building stuff for EPEL 9 is broken. :(

**Comment by praiskup** [comment]
I mean, can the 'epel-next-9' thing be delayed?  I agree that we want to have epel-9 working.

**Comment by Conan-Kudo** [comment]
Maybe? I don't think so, though. People are building for EPEL Next 9: https://bodhi.fedoraproject.org/releases/EPEL-9N

**Comment by hroncok** [comment]
> People are building for EPEL Next 9

Those are [mostly mistakes](https://pagure.io/releng/issue/10463) or [builds that happened before the EPEL 9 announcement](https://pagure.io/releng/issue/10456) though. Currently both EPEL 9 and EPEL 9 Next build against c9s, so there is no legitimate reason to build for EPEL 9 Next.

**Comment by praiskup** [comment]
... or do we want to delay this when probably nobody wants this, yet?

**Comment by Conan-Kudo** [comment]
I'm syncing it now, one sec

**Comment by Conan-Kudo** [comment]
@praiskup Done!

**Comment by praiskup** [comment]
LGTM, thanks.

**Comment by praiskup** [comment]
OK, merging soon as well, so it can be tested before the release.  Feel free to send follow-up patches.

**Pull request merged**